sound/soc/sof/ipc4-priv.h

```diff
@@ -48,4 +48,14 @@ extern const struct sof_ipc_pcm_ops ipc4_pcm_ops;
 
 int sof_ipc4_set_pipeline_state(struct snd_sof_dev *sdev, u32 id, u32 state);
 
+/**
+ * struct sof_ipc4_dx_info: ipc4 core/Dx mask data
+ * @core_mask: core mask
+ * @dx_mask: Dx mask
+ */
+struct sof_ipc4_dx_info {
+	u32 core_mask;
+	u32 dx_mask;
```

Comment:
this is very confusing, please describe more.

is core_mask the active cores, the managed cores? Why do we need a new definition?

Likewise Dx means 1 for D0 and 0 for D3? What about D0ix for core0?